tests/scripts/oversubscribable_empty.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+# do nothing here, so output file remains empty
```

Comment:
OPT: You can likely change your test to avoid the need of having an empty bash script, using some standard commands for example.